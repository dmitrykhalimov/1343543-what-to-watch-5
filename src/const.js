export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
