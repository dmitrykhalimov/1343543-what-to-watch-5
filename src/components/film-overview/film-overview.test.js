import React from "react";
import renderer from "react-test-renderer";
import FilmOverview from "./film-overview";
import {TEST_MOCKS} from "../../const";

describe(`FilmOverview`, () => {
  it(`Should FilmOverview render correctly`, () => {
    const tree = renderer
      .create(
          <FilmOverview
            film={TEST_MOCKS.film}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

