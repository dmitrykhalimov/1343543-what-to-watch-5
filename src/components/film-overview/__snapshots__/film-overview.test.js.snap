// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmOverview Should FilmOverview render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      1
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        
      </span>
      <span
        className="movie-rating__count"
      >
        1
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        
      </strong>
    </p>
  </div>,
]
`;
